core/commands/status_bar.py

```diff
@@ -27,19 +27,29 @@ def on_post_save(self, view):
 update_status_bar_soon = False
 
 
+def view_is_transient(view):
```

Comment:
The description of this PR can go into this function's docstring

Reply:
🙄 	I skipped reading the mail after seeing the ❤️ 	above.

Reply:
Only noticed that this could be improved after sending the approval comment